app/src/main/java/eu/kanade/tachiyomi/data/database/DbOpenCallback.kt

```diff
@@ -22,7 +22,7 @@ class DbOpenCallback : SupportSQLiteOpenHelper.Callback(DATABASE_VERSION) {
         /**
          * Version of the database.
          */
-        const val DATABASE_VERSION = 26
+        const val DATABASE_VERSION = 31
```

Comment:
just an fyi i will drop this down and combine some of the if statements.  It really only needed incremented because we were on a dev build